etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/metadata/partner/PartnerMetadataOrchestrator.java

```diff
@@ -143,43 +144,45 @@ public Optional<PartnerMetadata> getMetadata(String receivedSubmissionId)
 
         PartnerMetadata partnerMetadata = optionalPartnerMetadata.get();
         var sentSubmissionId = partnerMetadata.sentSubmissionId();
-        if (metadataIsStale(partnerMetadata) && sentSubmissionId != null) {
-            logger.logInfo(
-                    "Receiver name not found in metadata or delivery status still pending, looking up {} from RS history API",
-                    sentSubmissionId);
-
-            String receiver;
-            String rsStatus;
-            String rsMessage = "";
-            String timeDelivered;
-            try {
-                String bearerToken = rsclient.getRsToken();
-                String responseBody =
-                        rsclient.requestHistoryEndpoint(sentSubmissionId, bearerToken);
-                var parsedResponseBody = getDataFromReportStream(responseBody);
-                receiver = parsedResponseBody[0];
-                rsStatus = parsedResponseBody[1];
-                rsMessage = parsedResponseBody[2];
-                timeDelivered = parsedResponseBody[3];
-            } catch (ReportStreamEndpointClientException | FormatterProcessingException e) {
-                throw new PartnerMetadataException(
-                        "Unable to retrieve metadata from RS history API", e);
-            }
 
-            var ourStatus = ourStatusFromReportStreamStatus(rsStatus);
+        if (!metadataIsStale(partnerMetadata) || sentSubmissionId == null) {
```

Comment:
Is this what we want? Will sentSumbissionId ever get updated in this case?

Reply:
I just inverted the if-statement to avoid nesting, didn't change the actual logic

Reply:
Reverted this change for now to reduce noise in this PR